models/articles.models.js

```diff
@@ -28,3 +28,25 @@ exports.selectArticles = () => {
       return result.rows;
     });
 };
+
+exports.selectArticleCommentsById = (articleID) => {
+  return db
+    .query(
+      `SELECT comments.comment_id, comments.votes, comments.created_at, comments.author, comments.body, articles.article_id
+        FROM comments
+        RIGHT JOIN articles
+        ON articles.article_id = comments.article_id
+        WHERE articles.article_id = $1
+        ORDER BY created_at DESC;`,
```

Comment:
so this is a good idea but if there are no comments we're still getting information back that we don't require - is there another way we could check the the article_id exists and if it does exist but the rowCount is 0 send back empty array...

Reply:
thanks Heather, I've made the changes, do I have to hold fire and submit another pr to get it authorized?